docs/guide/snaps-concepts.md

```diff
@@ -0,0 +1,44 @@
+# Concepts
+
+[[toc]]
+
+## Accounts and Key Management
+
+Snaps contain functionality that allows you to manage users' private keys, with their approval. This capability comes with great responsiblity on the part of the developer: misplaced or stolen private keys will lead to a complete loss of funds for the users of the Snap. Our team has developed some guidelines to help you assess the feasibility and sanity of your Snap project.
```

Comment:
"Snaps API" is singular so it should be "Snaps contain**s** functionality"

then, "for the users of the snap." (lowercase) and "your snap project." (also lowercase)